src/reducers/counter.js

```diff
@@ -0,0 +1,12 @@
+const counterReducer = (state = 0, action) => {
+  switch (action.type) {
+    case 'INCREMENT':
+      return state + 1;
+    case 'DECREMENT':
+      return state - 1;
+    default:
+      return state;
+  }
+};
+
+export default counterReducer;
```

Comment:
Here and in the other files there is no need to assign some value to `const` to than export it as default anyway, it may be shorten by (in this case):
`export default (state = 0, action) => { ...`